internal/querycoord/meta.go

```diff
@@ -735,12 +645,20 @@ func (m *MetaReplica) sendSealedSegmentChangeInfos(collectionID UniqueID, change
 
 	messageIDInfos, err := queryStream.ProduceMark(msgPack)
 	if err != nil {
-		log.Error("updateGlobalSealedSegmentInfos: send sealed segment change info failed", zap.Int64("collectionID", collectionID), zap.Error(err))
-		return nil, nil, err
+		log.Error("sendSealedSegmentChangeInfos: send sealed segment change info failed", zap.Int64("collectionID", collectionID), zap.Error(err))
+		return nil, err
+	}
+	// len(messageIDs) = 1
+	messageIDs, ok := messageIDInfos[queryChannel]
```

Comment:
Add a constraint checking for length of messageIDs.

Reply:
fixed